app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class ="forms">
+  <h2>Sign up</h2>
+
+   <%= form_for(resource, as: resource_name, url: registration_path(resource_name), data: { turbo: false }) do |f| %>
+   <%= render "devise/shared/error_messages", resource: resource %>
+  
+    <div class= "mb-3 col-md-6">
+      <%= f.label :name, class:'label' %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name", class:"form-control", required: true %>
+    </div>
+
+    <div class= "mb-3 col-md-6">
+      <%= f.label :email, class:'label' %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class:"form-control", required: true %>
+    </div>
+
+    <div class= "mb-3 col-md-6">
+      <%= f.label :password, class:'label' %>
+      <%= f.password_field :password, autocomplete: "new-password", minlength: 6, class:"form-control"%>
+    </div>
+  
+    <div class= "mb-3 col-md-6">
+      <%= f.label :password_confirmation, class:'label' %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password",
+       required: true, class:"form-control"%>
+    </div>
+    
+    <div class= "mb-3 col-md-6">
+     <%= f.submit "Sign Up", class:"btn btn-primary", id:"post-btn" %>
+    </div>
+  <% end %>
+  
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- [OPTIONAL] It would be nice to improve the styling of the signup and the login pages. Having good designs give a good impression to recruiters and other developers. It shows that you pay attention to details. Kindly work on the styling of the splash page in both the development and production stage. It would be nice to implement this change 😎 💻

| Your Design - Signup Page  | Your Design - Login | Design Template - Signup | Design Template - Login | 
| ------------------------------ | ---------------------- | ---------------------------- | --------------------------- |
| ![image](https://user-images.githubusercontent.com/77106962/235506142-2603a2b0-b927-45cf-b7b3-f0e7bf978b17.png)  | ![image](https://user-images.githubusercontent.com/77106962/235506267-f057c975-2eac-4d6d-8b20-271e10299099.png) | ![image](https://user-images.githubusercontent.com/77106962/235506489-7f5a586d-7e57-4b4c-a786-9fd509263ddc.png) | ![image](https://user-images.githubusercontent.com/77106962/235506774-4c5cef0f-3b35-4936-a1b2-db1eef2426d7.png) |